sound/soc/sof/intel/cnl.c

```diff
@@ -266,5 +266,6 @@ const struct sof_intel_dsp_desc cnl_chip_info = {
 	.ipc_ack = CNL_DSP_REG_HIPCIDA,
 	.ipc_ack_mask = CNL_DSP_REG_HIPCIDA_DONE,
 	.ipc_ctl = CNL_DSP_REG_HIPCCTL,
+	.rom_init_timeout	= 300,
```

Comment:
paranoia question: I don't have the code in front of me, but is this rom_init_timeout going to interfere with the 300ms timeout for IPC? Or is this a different definition completely?

Reply:
@plbossart I cant imagine a scenario where this could happen. At this stage of FW boot, we shouldnt have sent out any IPC's to the DSP from the host. So, it should be safe.